### PR TITLE
List photos

### PR DESCRIPTION
**Pull request by molinajulian**
## Summary 
It is allowed to show the photos of an album bought by a user.

## Trellocard
https://trello.com/c/W6bx97M7

### CODE REVIEW

File: Training_node/first_project/app/controllers/users.js [hunk 1]
```diff
@@ -178,3 +180,40 @@ exports.getAllAlbums = (req, res, next) => {
...
+      res.status(200);
+      res.send({ photos: result });
+      res.end();
+    });
+  } else {
+    next(errors.defaultError(msg));
+  }
+};
+exports.getPhotos = (req, res, next) => {
+  const idAlbum = parseInt(req.url.split('/')[3]);
```
Comment (gonziesc) [inline Training_node/first_project/app/controllers/users.js]: req.params.id

File: Training_node/first_project/test/users.js [hunk 2]
```diff
@@ -368,3 +376,48 @@ describe('/users/:user_id/albums GET', () => {
...
 });
+
+describe('/users/albums/:id/photos GET', () => {
+  beforeEach(() => {
+    nock(config.common.urlRequests.base)
+      .get('/photos?albumId=2')
+      .reply(200, onePhoto);
+  });
+  it('should be successful', done => {
+    successfulLogin().then(res => {
```
Comment (gonziesc) [inline Training_node/first_project/test/users.js]: do we have a test for admins?

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: Training_node/first_project/test/users.js (hunk 2 above)
Comment (molinajulian) [reply Training_node/first_project/test/users.js]: Yes! The function sucessfulLogin() if it dont receive params (email), log in with admin.

File: Training_node/first_project/app/controllers/albums.js [hunk 3]
```diff
@@ -34,6 +34,7 @@ exports.getById = (req, res, next) => {
               res.end();
             });
           } else {
+            logger.info('ERRR');
```
Comment (gonziesc) [inline Training_node/first_project/app/controllers/albums.js]: delete this

**Review by gonziesc — CHANGES_REQUESTED** [review]

**Review by gonziesc — APPROVED** [review]

**Pull request merged**